### PR TITLE
(WIP) adding additional logs to fast integration tests (part 2)

### PR DESCRIPTION
**Pull request by tillknuesting**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: tests/fast-integration/kcp/client.js [hunk 1]
```diff
@@ -280,21 +280,37 @@ class KCPWrapper {
...
+
+      if (objReconciliationsOperationsLength === 0) {
+        console.log(`\nNo reconciliation operations found`);
+        return;
+      }
+      console.log(`\nNumber of reconciliations operations: ${objReconciliationsOperationsLength}`);
+
+      // using only last three operations
+      const lastObjReconciliationsOperations = objReconciliationsOperations
+          .slice(objReconciliationsOperations.length - 3, objReconciliationsOperations.length);
```
[the remark this answers is not in the document]
Comment (wozniakjan) [reply tests/fast-integration/kcp/client.js:301]: I think the first argument could be `max(0, objReconciliationsOperations.length - 3)` to cover this corner case and we are fine

**Review by wozniakjan — DISMISSED** [review]
/lgtm

**Review by wozniakjan — DISMISSED** [review]
/lgtm

**Review by wozniakjan — APPROVED** [review]
